app/base.scss

```diff
@@ -0,0 +1,61 @@
+*{
+  padding: 0px;
+  width: 100%;
+  font-size: 1em;
+  font-weight: normal;
+  color: #ff4;
```

Comment:
Don't hardcode colors in, just use your variables. Since your variable ``$brand-focus`` is also a similar yellow, I'd use there instead.